pyomo/opt/tests/base/test_solver.py

```diff
@@ -24,18 +24,18 @@
 old_tempdir = TempfileManager.tempdir
 
 
-class TestSolver1(pyomo.opt.OptSolver):
+class DummySolver1(pyomo.opt.OptSolver):
```

Comment:
Just a suggestion: I think it's more common to refer to these as "mock" objects in Python. I'm fine with `DummySolver` but we may want to consider `MockSolver`